test/integration/targets/win_get_url/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
-test_win_get_url_path: '{{win_output_dir}}\win_get_url'
+test_win_get_url_path: '{{win_output_dir}}/win_get_url'
```

Comment:
I would be using backslashes, as this is what we recommend our users.

Reply:
Fixed